repos/config/detail/http_archive.bzl

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "update_attrs")
+
+_archive_attrs = {
+    "url": attr.string(
+        doc = "URI of the archive.",
+    ),
+    "strip_prefix": attr.string(
+        doc =
+            "Path prefix to be stripped.",
+    ),
+    "sha256": attr.string(
+        doc =
+            "Hash of the commit to be checked out.",
+    ),
+    "build_files": attr.label_keyed_string_dict(
+        doc =
+            "Same as in native rule.",
```

Comment:
@kilian-funk the native rule is [just one `build_file`](https://bazel.build/rules/lib/repo/http#http_archive-build_file), no? Maybe explain this one is instead a dict of `build_files` with a link to the native rule docs on the `build_file` argument

Reply:
Definitely. Will fix it.